src/app/containers/SocialEmbed/index.jsx

```diff
@@ -72,6 +73,24 @@ const SocialEmbedContainer = ({ blocks }) => {
     href,
   });
 
+  const socialEmbed = (
+    <CanonicalSocialEmbed
+      provider={provider}
+      service={service}
+      oEmbed={oEmbed}
+      fallback={fallback}
+      skipLink={skipLink}
+      caption={caption}
+    />
+  );
+
+  const enrichedSocialEmbed =
+    provider === 'twitter' ? (
+      <EnrichTweet>{socialEmbed}</EnrichTweet>
+    ) : (
+      socialEmbed
+    );
```

Comment:
- This is called enrichedSocialEmbed, but could return an 'unenriched' social embed.
- This shouldn't be necessary, because our hook checks whether `twttr` is on the global object and it will only be if there's a Social Embed from Twitter on the page.

Reply:
I'm happy to remove the conditional, it does though add an uneccessary `useEffect` to all social embeds that is avoided with this check. I don't know if adding lots of `useEffects` can harm the performance of the page; I imagine it's negligible?

Reply:
Yes, it's more what's running in the callback as opposed to the useEffect itself. The if-statement (inside the useEffect) guards against executing Twitter's logic unless it's needed.